src/riak_kv_ts_api.erl

```diff
@@ -78,6 +79,8 @@ api_call_to_perm(query_explain) ->
     "riak_ts.query_explain";
 api_call_to_perm(describe_table) ->
     "riak_ts.describe_table";
+api_call_to_perm(query_delete) ->
+    "riak_ts.query_delete";
```

Comment:
I would argue that this should converge onto "riak_ts.delete", as it is done for `query_insert`.